gridData/tests/test_dx.py

```diff
@@ -21,6 +21,16 @@ def test_read_dx():
     assert_equal(g.delta, np.ones(3))
     assert_equal(g.origin, np.array([20.1, 3., -10.]))
 
+def test_read_dxgz():
```

Comment:
We should be able to just parametrize the previous test over the uncompressed and compressed file – reduced code duplication.